tests/ng2web/src/app/room-ngrx/room-ngrx.list.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Room, Account, FireLoopRef } from '../shared/sdk/models';
+// import { AccountApi, RoomApi, LoggerService, RealTime } from '../shared/sdk/services';
+import { AccountApi, RoomApi, LoggerService, RealTime, SDKModels } from '../shared/sdk/services';
+import { Router } from '@angular/router';
+import { Subscription } from 'rxjs';
+import { LoopBackConfig } from '../shared/sdk/lb.config';
+LoopBackConfig.setBaseURL('http://127.0.0.1:3002');
+
+import * as io from 'socket.io-client';
```

Comment:
Why is the socket io client directly imported in here?.... It shouldn't be like that, though I don't see any implementation using the io library within the component class, so not sure why is this imported in here....